fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PipeAnalyzer.java

```diff
@@ -188,6 +193,12 @@ public static void analyze(CreatePipeStmt stmt, ConnectContext context) {
             ErrorReport.reportSemanticException(ErrorCode.ERR_BAD_PIPE_STATEMENT, "only support FileTableFunction");
         }
 
+        if (!stmt.getPipeName().getDbName().equalsIgnoreCase(insertStmt.getTableName().getDb())) {
```

Comment:
at line 185:
We can give more error message info, like: `must be a vanilla select statement. Aggregation, order by clause, limit cluase are not supported yet.`

Reply:
Updated.